app/chains/userToCourseConnector/userToCourseConnector.dao.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+var request = require('request-promise');
+var Promise = require('bluebird');
+var config = require('config');
+var responseHandler = require('../../utils/response.handler');
+var errorHandler = require('../../utils/error.handler');
+var utils = require('../../utils/utils');
+
+var url = config.API_URL + 'userToCourseConnector';
+
+exports.createUserToCourseConnector = function(userToCourseConnector) {
+    var options = {
+        resolveWithFullResponse: true,
+        uri: url,
+        method: 'POST',
+        json: userToCourseConnector,
+        gzip: true
+    };
+
+    return request(options)
+        .then(responseHandler.parsePost)
+        .catch(errorHandler.throwDREAMSHttpError);
+};
+
+exports.getUserToCourseConnectorById = function(id) {
+    var options = {
+        resolveWithFullResponse: true,
+        uri: url + '/' + id,
+        method: 'GET',
+        gzip: true,
+        json: true
+    };
+
+    return request(options)
+        .then(responseHandler.parseGet)
+        .catch(errorHandler.throwDREAMSHttpError);
+};
+
+exports.getUserToCourseConnectors = function(query) {
+    var options = {
+        resolveWithFullResponse: true,
+        uri: url + utils.getQueryByObject(query),
+        method: 'GET',
+        gzip: true
+    };
+
+    return request(options)
+        .then(function(value) {
+            return value;
```

Comment:
remove this. If it's from copy-paste programming, remove it where you founf it. This .then doesnt add anything.